docs/_layouts/default.html

```diff
@@ -35,7 +35,7 @@
 			</a>
 			<a href="{{ site.baseurl }}/" alt="Hubble Enterprise">
 				<img src="{{ site.baseurl }}/assets/images/logo-title.svg" class="project-logo" width="3rem" height="3rem">
-				<h1 class="project-name"><script type="text/javascript">document.write(gheHostname())</script> Statistics</h1>
+				<h1 class="project-name">{{ site.url | remove: "https://" }} Statistics</h1>
```

Comment:
Thank you 👍 ! Is there a way to handle `http://` URLs, too? Some GitHub Enterprise installations use just HTTP 😢

Reply:
Wasn’t aware of this, good catch!

Reply:
Testing it out.